packages/backend-common/config.d.ts

```diff
@@ -41,7 +41,7 @@ export interface Config {
       | string
       | {
           /** Address of the interface that the backend should bind to. */
-          address?: string;
```

Comment:
Seen as this is breaking, should it technically be a `minor` release?

Reply:
Not really breaking since it's all optional right? If people currently have `address` it will just be ignored, which it already was. If people currently have `host`, it's all good, unless it's not a string, but that would've already thrown an error.

Reply:
Ah sorry - I was under the impression that `additionalProperties` would cause this to break, but I misunderstood.